frontend/src/components/TopicCard/index.js

```diff
@@ -29,8 +29,8 @@ class ReactTopicCard extends Component {
     if (topic.authorizePermissionChange(currentUser)) classname += ' yourTopic'
 
     return (
-      <Draggable handle=".metacodeImage" defaultPosition={{x: 100, y: 100}}>
-        <div className="showcard mapElement mapElementHidden">
+      <Draggable handle='.metacodeImage' defaultPosition={{x: 100, y: 100}}>
+        <div className='showcard mapElement mapElementHidden'>
```

Comment:
is react ok with single quotes on props? i am if it is

Reply:
yep